cmd/api/book/book.go

```diff
@@ -0,0 +1,34 @@
+package book
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+const PriceMax = 9999.99 //max value to field price on db, set to: numeric(6,2)
+
+type Book struct { //IS IT POSSIBLE TO MOVE JSON TAGS TO HTTP PACKAGE??
+	ID        uuid.UUID `json:"id"`
+	Name      string    `json:"name"`
+	Price     *float32  `json:"price"`
+	Inventory *int      `json:"inventory"`
+	CreatedAt time.Time `json:"-"`
+	UpdatedAt time.Time `json:"-"`
+	Archived  bool      `json:"archived"`
+}
```

Comment:
We should, but I want to finish this refactor first, we can deal with that later. I will make sure we have a ticket to address it.

Reply:
https://trello.com/c/scwnaydr/56-refactor-define-request-types